packages/11ty/_plugins/figures/sequence/index.js

```diff
@@ -12,12 +12,12 @@ const Annotation = require('../annotation')
  * @return {Sequence}
  */
 module.exports = class Sequence {
-  constructor(figure, sequence) {
+  constructor(figure, sequence, files) {
     const { iiifConfig } = figure
     this.behavior = sequence.behavior
     this.dir = sequence.id
     this.figure = figure
-    this.files = getSequenceFiles(sequence, iiifConfig)
+    this.files = getSequenceFiles(sequence, iiifConfig, files)
```

Comment:
@anderspollack this makes sense and if we make gathering the `files` a concern of the factory then it can pass them to the `Sequence` constructor, which may make both easier to test with a fixture for the files.